### ISSUE TITLE
Bug: Text for reactions on highlights fail all colour contrast checks

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2289]
### Describe the bug

Text for reactions on highlights fail all colour contrast checks

### Steps to reproduce

1. Go to https://app.opensauced.pizza/feed
2. Look at any highlight with a reaction. The orange color on the background completely fails colour contrast checks.

![CleanShot 2023-12-08 at 10 34 01](https://github.com/open-sauced/app/assets/833231/d49319b9-6ffb-44b8-814d-b7ed23642c82)

I'd recommend going with a dark text for the reaction counts.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by nickytonline** [comment]
We should remobe so it looks like this instead.

https://github.com/open-sauced/app/blob/c504b67c7e99e1f5ff129dbcb29105c469cad440/components/molecules/ContributorHighlight/contributor-highlight-card.tsx#L658

```diff
                 <Emoji
-                  className="text-xs text-orange-500 md:text-sm"
+                  className="text-xs md:text-sm"
                   text={`:${getEmojiNameById(emoji_id)}: ${reaction_count}`}
                 />
```

![CleanShot 2023-12-08 at 10 39 21](https://github.com/open-sauced/app/assets/833231/34b3328e-f900-4175-867f-ab9dfc536363)

**Comment by Himanshuch8055** [comment]
Are you still working on it, @ItIsOHM

**Comment by nickytonline** [comment]
@ItIsOHM, hope you feel better. @Himanshuch8055, I've assigned this to you.
